packages/cli/lib/prompts/localDevUploadPrompt.js

```diff
@@ -0,0 +1,49 @@
+const readline = require('readline');
+const chalk = require('chalk');
+// const { promptUser } = require('./promptUtils');
+const { i18n } = require('../lang');
+
+const i18nKey = 'lib.prompts.localDevUploadPrompt';
+
+const makeUploadPrompt = message => {
+  return () => {
+    const rl = readline.createInterface({
```

Comment:
I'm not exactly sure where it's coming from, but it seems like my inputs for each prompt are not clearing between prompts. You can reproduce this by getting the prompt to show up several times in a row. The first time when I type "n" I see "n" in the terminal. The second time I see "nn", then "nnn", and so on.

Somewhat related, I'm able to type `q` and the command immediately exits (I think that's WAD), but the cleanup logs show on the same line as the upload prompt. Maybe we want to exit the prompt before logging the cleanup commands? Here's an example:
<img width="592" alt="image" src="https://github.com/HubSpot/hubspot-cli/assets/6654014/5eb74281-ee1d-45db-968d-dbdb12f6f758">

Reply:
hmmm that is...not ideal. looking into it

Reply:
OK, I fixed the issue where keypresses were logging twice. Also opted to have `q` _not_ exit the command when the prompt is open. Felt a little weird since you can theoretically type anything into the prompt.

It now cleanly shuts down (and tells you to upload to see your config changes) when you press control C with the prompt open